buildSrc/src/main/kotlin/Versions.kt

```diff
@@ -1,6 +1,6 @@
 object Versions {
     const val dokka = "0.10.1"
-    const val kotlin = "1.4.21-2"
+    const val kotlin = "1.4.10"
```

Comment:
js benchmarking failing to compile with anything higher, opened a ticket: https://github.com/Kotlin/kotlinx-benchmark/issues/28

Reply:
disabled js benchmark target and bumped to 1.4.30